core/src/options/outer_from.rs

```diff
@@ -91,5 +92,17 @@ impl ParseData for OuterFrom {
 
     fn validate_body(&self, errors: &mut crate::error::Accumulator) {
         self.container.validate_body(errors);
+        if let Some(attrs) = &self.attrs {
+            if self.forward_attrs.is_none() {
+                let container_name = match &self.container.data {
+                    Data::Enum(_) => "enum",
+                    Data::Struct(_) => "struct",
+                };
```

Comment:
I'm surprised this hasn't been used before; might be worth adding a function to `Data` for it?

Reply:
Perhaps check out [strum](https://crates.io/crates/strum) for a quick derive macro approach to giving enums display names

Reply:
This isn't actually getting the name, it's just the word "enum" or "struct"